packages/blockchain-utils-linking/src/__tests__/solana.test.ts

```diff
@@ -0,0 +1,54 @@
+import { MessageSignerWalletAdapterProps } from '@solana/wallet-adapter-base';
+import { SOLANA_MAINNET_CHAIN_REF, SolanaAuthProvider } from '../solana'
+import { Keypair } from '@solana/web3.js';
+import { Buffer } from 'buffer';
```

Comment:
There is no buffer in a browser. Please, use `Uint8Array`.

Reply:
Oh, now I see it is a test. Still, we have no Buffer policy. `Uint8Array` and `uint8arrays` package are your friends here.

Reply:
for sure!  thanks for pointing this out :)